src/status_im/contexts/chat/messenger/messages/navigation/view.cljs

```diff
@@ -103,15 +104,20 @@
        :banner-opacity banner-opacity
        :top-offset     navigation-view-height}]]))
 
-(defn f-view
+(defn view
   [{:keys [distance-from-list-top chat-screen-layout-calculations-complete?]}]
   (let [{:keys [chat-id chat-type] :as chat} (rf/sub [:chats/current-chat-chat-view])
         all-loaded?                          (reanimated/use-shared-value false)
         all-loaded-sub                       (rf/sub [:chats/all-loaded? chat-id])
         top-insets                           (safe-area/get-top)
         top-bar-height                       messages.constants/top-bar-height
-        navigation-view-height               (+ top-bar-height top-insets)]
-    (reanimated/set-shared-value all-loaded? all-loaded-sub)
+        navigation-view-height               (+ top-bar-height top-insets)
+        reached-top?                         (messages.worklets/use-messages-scrolled-to-top
+                                              distance-from-list-top
+                                              top-bar-height)
+        button-background                    (if reached-top? :photo :blur)]
+    (rn/use-effect (fn [] (reanimated/set-shared-value all-loaded? all-loaded-sub))
```

Comment:
hi @flexsurfer, Do we have `use-sub-as-shared-value` feature?

Reply:
yes, I think we don' have this feature yet.
Related discussion - https://discord.com/channels/1103692771585433630/1172149795956203530/1210158688699027507

Reply:
> hi @flexsurfer, Do we have `use-sub-as-shared-value` feature?

not yet :( https://github.com/status-im/status-mobile/issues/18946